docs/security-considerations.rst

```diff
@@ -284,7 +284,7 @@ field of a ``struct`` that is the base type of a dynamic storage array.  The
 
         function allocate(uint _newMaps) public {
             for (uint i = 0; i < _newMaps; i++)
-                array.push();
+                array.push(uint(0));
```

Comment:
Does just `push()` not work?

Reply:
It complains with 
`Error: Wrong argument count for function call: 0 arguments given but expected 1.`

Maybe increasing version number would help

Reply:
`push()` without arguments was introduced in 0.6.0 if we trust to Changelog :)
I'll update minimum version for samples that have this call